markerclustererplus/src/markerclusterer.js

```diff
@@ -361,6 +377,18 @@ ClusterIcon.prototype.getPosFromLatLng_ = function (latlng) {
   return pos;
 };
 
+/**
+ * Sync the eventDiv_ size and position with the display div_
+ */
+ClusterIcon.prototype.syncDivStyle = function (vars) {
+  if (vars) {
+    Object.assign(this.eventDiv_, vars);
+  }
+  if (this.eventDiv_ && this.div_) {
+    this.eventDiv_.style.cssText = this.div_.style.cssText;
```

Comment:
The overlayDiv, `this.div_`, is positioned and resized using CSS. By copying the cssText we copy the same height, width, z-index, etc. without having to specify each.